core/events_inc.php

```diff
@@ -89,6 +89,10 @@
 	'EVENT_MANAGE_USER_DELETE' => EVENT_TYPE_EXECUTE,
 	'EVENT_MANAGE_USER_PAGE' => EVENT_TYPE_EXECUTE,
 
+	'EVENT_MANAGE_PROJECT_USER_CREATE' => EVENT_TYPE_CHAIN,
+    'EVENT_MANAGE_PROJECT_USER_UPDATE' => EVENT_TYPE_CHAIN,
+    'EVENT_MANAGE_PROJECT_USER_DELETE' => EVENT_TYPE_CHAIN,
```

Comment:
Wondering why you are specifying a CHAIN event type, as you are not using the event's return value. Maybe you meant to use EVENT_TYPE_EXECUTE ?